### ISSUE TITLE
Frame obligations as positive rights?

### ISSUE THREAD

**Issue by adamhjk** [adamhjk/community-compact#7]
Could we frame the obligation of commercial users as a positive right rather than a negative one?

**Comment by adamhjk** [comment]
I think it's an interesting question. The more thought I've put into this, the more convinced I get that there are several different compacts hiding in here, with different choices for community norms. The freeloaders idea is an interesting one, in part because I don't think anyone has ever done it. 

I've been looking into what I've come to call the "Open Source Product" model, which is the one that is the closest to what I describe in the original community compact. In that model, it feels like the question regarding the FREELOADERS file is really centered around wether or not the company itself is going to produce the 100% free build. If it is, and it retains the same trademarks, then I think you are going to need the FREELOADERS.

If it's not, then I think you're explicitly expecting that there will be a downstream build of the software, using different trademarks. At that point, you don't need FREELOADERS, because the company and the community have made their respective beds.

**Comment by tieguy** [comment]
Yeah, modularity is always something that comes up in these sorts of discussions. (A|L|GPL, Creative Commons variations, etc.) It often makes sense given the vast modularity in projects out there.

Since this is still pretty early/experimental/informal, might be worth simply having a "design notes" or "other options" text file that talks about any options you've thought about but not put in the current main document. That could give you some flexibility to think out loud, and not lose big/interesting ideas like FREELOADERS. (Obviously to some extent the git history also gives you that, but surfacing it through a top-level, quasi-maintained doc means you won't have to re-explain to everyone who doesn't want to go through the history.)
